dispersion_protocol/dispersion.xml

```diff
@@ -0,0 +1,184 @@
+      <!-- BEGIN of dispersion device commands-->
+      <entry value="30010" name="MAV_CMD_DO_CONFIG_DISPERSION_PARAMS" hasLocation="false" isDestination="false">
+        <description>This command provides the necessary run time information for the dispersion device to support a specific configuration associated with the target dispersion profile.</description>
+        <param index="1" label="Dispersion Type" enum="MAV_DISPERSION_TYPE">The type of dispersion this device supports. This indicates target units as outlined in enum</param>
+        <param index="2" label="Reference Speed" units="m/s">The target speed for which any target dispersion rate applies, since the rate should change with speed</param>
+        <param index="3" label="Dispersion Rate Min" units="liters/min or kg/min">Minimum dispersion rate the current configuration supports</param>
+        <param index="4" label="Dispersion Rate Max" units="liters/min or kg/min">Maximum dispersion rate the current configuration supports</param>
+        <param index="5" label="Pressure Min" units="Pa">If this is a spray dispersion system, minimum pressure the current configuration supports</param>
+        <param index="6" label="Pressure Max" units="Pa">If this is a spray dispersion system, maximum pressure the current configuration supports</param>
+        <param index="7" label="Target Fill Level" units="liters or kg">Target payload amount for device. This will be ignored if the capability flags do not indicate fill level detection support</param>
+      </entry>
+      <entry value="30011" name="MAV_CMD_DO_SET_DISPERSION_RATE" hasLocation="false" isDestination="false">
+        <description>Command to provide real time adjustment to dispersion device output for a given dispersion profile configured using MAV_CMD_CONFIG_DISPERSION_DEVICE_PARAMS.</description>
+        <param index="1" label="Timestamp Ns" units="ns">The unix timestamp for message creation</param>
+        <param index="2" label="Dispersion Type" enum="MAV_DISPERSION_TYPE">The type of dispersion this device supports. This indicates units as outlined in enum</param>
+        <param index="3" label="Target Dispersion Rate" units="liters/min or kg/min">Target dispersion rate for the system</param>
+        <param index="4">Empty</param>
+        <param index="5">Empty</param>
+        <param index="6">Empty</param>
+        <param index="7">Empty</param>
+      </entry>
+      <entry value="30012" name="MAV_CMD_DO_SET_DISPERSION_LOCK" hasLocation="false" isDestination="false">
```

Comment:
When would you send this?

Reply:
Ah, I see from the other doc. Ultimately the messages/commands in the XML are the source of truth. So something like "should only be used in emergency might reasonably be included in the description.